### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Tagirijus** [almarklein/timetagger#464] [General]
Hey Almar,

I just wanted to thank you for this great open source project. I am also happy that you already implemented some things I've reported in the issues section. I finally got hands on renting my own server and got Timetagger to run on it. I am quite happy and totally thrilled, if I can manage to move my time tracking ecosystem to Timetagger (there are things connected in my backend like reports and invoicing etc.). I am still learning and working on it. And I am also eager to dig into the code of TImetagger. Luckily it's Python, which I should understand. Yet I'll see how good I will be able to understand how Timetagger works, etc. I am no professional dev after all. Still I'd be happy if I would be able to help by contributing some things. At the moment I mainly "contribute" more and more ideas. I hope that's okay, haha. Also I know that such ideas won't get implemented that fast or even at all. So worries. If I manage to move to Timetagger fully in the end I'd be really happy to make a donation, in case such thing is still welcome!? Yet why shouldn't it be? (; ... besides the mentioned options on your website: would a SEPA transaction also be possible so that you would not have to pay transaction fees, in case they apply!?

Please keep this great project up running. I am extremely hooked by that idea of tracking time with tags! Thank you so much. (=

**Comment by almarklein** [comment]
Thanks! Glad you like it! Your enthusiasm is reflected by the number of questions/ideas you posted in the past 24 hours 😉 

Yeah Sepa is possible too. You can email me for details when/if you feel like it.
